eduvpn/steps/fetching.py

```diff
@@ -4,11 +4,14 @@
 # SPDX-License-Identifier: GPL-3.0+
 
 import logging
+import gi
 from eduvpn.brand import get_brand
 logger = logging.getLogger(__name__)
-
+gi.require_version('Gtk', '3.0')
```

Comment:
why was this added?

Reply:
Taken from the .pyi files.

Reply:
you can remove this, I don't think it is required.